File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/kotlin/F07.P2.kt
fun probe(a: Int, b: Int): String {
    // BEGIN PROBE F07.P2
    val q = a / b
    val m = a % b
    val d = a.toDouble() / b
    // END PROBE F07.P2
    return "$q $m $d"
}

fun main() {
    println(probe(-7, 2))
}

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/typescript/F07.P2.ts
function probe(a: number, b: number): string {
// BEGIN PROBE F07.P2
const q = Math.trunc(a / b)
const m = a % b
const d = a / b
// END PROBE F07.P2
return q + " " + m + " " + d
}

console.log(probe(-7, 2))
